docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
+       key: token
+
+   connInfoSecretTarget:
+       name: cassandra-secret
+
+   userConfig:
+       migrate_sstableloader: true
+       service_log: true
+
+   project: PROJECT_NAME
+   cloudName: google-europe-west1
+   plan: startup-4
+
+   maintenanceWindowDow: sunday
+   maintenanceWindowTime: 22:00:00
+   terminationProtection: true
+   ```
+
+1. Update the service by applying the configuration:
+
+   ```go
+   kubectl apply -f cassandra-sample.yaml
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```go
+   kubectl describe cassandra.aiven.io cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
+
+See the available configuration options in
+[Aiven Operator for Kubernetes®: Cassandra](https://aiven.github.io/aiven-operator/api-reference/cassandra.html).
+</TabItem>
+</Tabs>
+
+## Connect to the service{#connect-to-service}
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+1. Log in to the [Aiven Console](https://console.aiven.io/), and go to your
+   organization > project > Aiven for Apache Cassandra service.
+1. On the <ConsoleLabel name="overview"/> page of your service, click
+   **Quick connect**.
+1. In the **Connect** window, select a tool or language to connect to your service, follow
+   the connection instructions, and click **Done**.
+
+   ```bash
+   # Sample command for connecting with the cqlsh client
+
+   ./cqlsh --ssl -u USER_NAME -p PASSWORD HOST_NAME PORT_NUMBER
+   ```
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+Access your new service with the Apache Cassandra client using the environment variables
+assigned to Terraform outputs:
 
 ```bash
-avn service create demo-cassandra       \
-   --service-type cassandra             \
-   --cloud CLOUD_AND_REGION             \
-   --plan PLAN                          \
-   --project PROJECT_NAME
+./cqlsh --ssl --host=$CASSANDRA_HOST --port=$CASSANDRA_PORT --user=$CASSANDRA_USER --password=$CASSANDRA_PASSWORD
 ```
 
-:::note
+</TabItem>
+<TabItem value="3" label="cqlsh client">
+[Connect to your new service with CLI](/docs/products/cassandra/howto/connect-cqlsh-cli)
+using the
+[the `cqlsh` client](https://cassandra.apache.org/doc/stable/cassandra/tools/cqlsh.html).
+</TabItem>
+</Tabs>
 
-- To preview the full list of default flags, run `avn service create -h`.
-- To list the type-specific options, run `avn service types -v`.
+:::tip
+Discover more tools for connecting to Aiven for Apache Cassandra in
+[Connect to Aiven for Apache Cassandra®](/docs/products/clickhouse/howto/list-connect-to-service).
+:::
+
+## Create keyspaces and tables
+
+Use `cqlsh` to create a keyspace and a table where you can insert your data next.
+
+1. Create a keyspace, for example `library`:
+
+   ```sql
+   CREATE KEYSPACE IF NOT EXISTS library WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '3' };
```

Comment:
could you change the example to use NetworkTopologyStrategy? I want to sunset SimpleStrategy eventually, the replication part should look like this:
```
... REPLICATION = { 'class': 'NetworkTopologyStrategy', 'aiven': 3 }
```
where 'aiven' is the default datacenter name for Aiven for Cassandra services.